ESP8266/ESP8266.cpp

```diff
@@ -215,7 +220,7 @@ void ESP8266::_packet_handler()
     uint32_t amount;
 
     // parse out the packet
-    if (!_parser.recv(",%d,%d:", &id, &amount)) {
+    if (!_parser.recv(",%d,%" SCNu32 ":" , &id, &amount)) {
```

Comment:
Well, either way, the function's argument does not have significance as a uint32_t. I would keep the old string to be consistent with the rest of the driver.